specs/core/0_beacon-chain.md

```diff
@@ -789,6 +795,17 @@ def compute_domain(domain_type: DomainType, fork_version: Version=Version()) ->
     return Domain(domain_type + fork_version)
 ```
 
+### `compute_domain_wrapper_root`
```

Comment:
"domain wrapper" feels semantically wrong. It's not the domain that is being wrapped, it's the object root. Also the wrapping adds more than just the domain (in particular, it adds the uniqueness root).

What about `compute_signing_root`? (Also shorter name.)

Reply:
By uniqueness root, you mean the root of the object?

I would say `compute_signing_root` or something similar works.
Funny, this is intimately tied to the other conversation on what to call the output of this function

Reply:
> By uniqueness root, you mean the root of the object?

No, a different root where `uniqueness_root = hash_tree_root(uniqueness_data))` :) See https://github.com/ethereum/eth2.0-specs/issues/1387 which you suggested by handled in a separate PR for clarity.

Reply:
So IMO, `signing_root` is the correct name here. The problem is the collision with the signing root removed from SSZ. My concern is that it may be confusing to implementors (particularly if they are implementing both changes at once.) Do people that this is a problem?

An alternate name is `domain_wrapped_root` which is semantically correct.

As @djrtwo points out, the naming problem exists across the whole PR. Which is what I was getting at when I mentioned that "In general, I am not 100% satisfied with the names I've implemented and am open to suggestions."

Reply:
I'm okay with `signing_root` with a warning in the release. It is semantically correct and easy to discuss verbally.

If this were a phase 1 question after release of phase 0, I'd be a little bit more concerned.

Reply:
Alright, I've gone with `signing_root`. :) It is definitely the clearest solution.